main.css

```diff
@@ -81,14 +86,348 @@
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.role-year {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 0.5px;
+  top: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.project-details {
+  position: static;
+  width: 295px;
+  height: 76px;
+  left: 0.5px;
+  top: 92px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0.5px;
+}
+
+.skills {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.5px;
+  position: static;
+  height: 24px;
+  left: 0.5px;
+  top: 180.5px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: 12px 0.5px;
+  margin-block-start: 0.5px;
+  margin-block-end: 0.5px;
+  padding-inline-start: 0.5px;
+}
+
+li.skillset {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  position: static;
+  width: 53px;
+  height: 24px;
+  left: 0.5px;
+  top: 0.5px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 8px;
+}
+
+.skillset {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 12px;
+  top: 4px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.div-button {
+  position: static;
+  width: 295px;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 24px 0.5px;
+}
+
+.button {
+  position: static;
+  width: 100.5px;
+  height: 48px;
+  left: 12px;
+  top: 12px;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 10.5px;
+  font-family: "Poppins", sans-serif;
+}
+
+button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px;
+  width: 124px;
+  height: 48px;
+  left: 0.5px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  margin-left: 12px;
+}
+
+.proj-myself {
+  background: #fff;
+  width: 375px;
+}
+
+.my-title {
+  display: flex;
+  position: static;
+  width: 327px;
+  height: 10.5px;
+  left: 0.5px;
+  top: 0.5px;
+  padding: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40.5px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0.5px;
+}
```

Comment:
Please ensure you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please do this both the work project section and the About section.

## Your design:
![your_about](https://user-images.githubusercontent.com/10332499/148553738-616bdfed-63a8-40fd-810e-64e0a68f739f.PNG)
![your_skill](https://user-images.githubusercontent.com/10332499/148553762-69dda549-d736-49cf-96ab-857d1f24f7dc.PNG)
![your_work](https://user-images.githubusercontent.com/10332499/148553783-69e9c025-91e9-4f56-8c45-ad862ea58e96.PNG)

## Figma design:
![figma_about](https://user-images.githubusercontent.com/10332499/148553858-6527acb4-8565-48cc-89ff-391fcff0c1f2.PNG)
![figma_skill](https://user-images.githubusercontent.com/10332499/148553870-3a289126-3ca4-4ef6-bb61-ede3c15677b0.PNG)
![figma_work](https://user-images.githubusercontent.com/10332499/148553882-26fd4527-e45c-4513-badf-421c2006959e.PNG)